packages/image_picker/image_picker_platform_interface/lib/src/types/camera_delegate.dart

```diff
@@ -0,0 +1,49 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:cross_file/cross_file.dart';
+import 'package:flutter/foundation.dart' show immutable;
+
+import 'camera_device.dart';
+
+/// Options for [ImagePickerCameraDelegate] methods.
+///
+/// New options may be added in the future.
+@immutable
+class ImagePickerCameraDelegateOptions {
+  /// Creates a new set of options for taking an image or video.
+  const ImagePickerCameraDelegateOptions({
+    this.preferredCameraDevice = CameraDevice.rear,
+    this.maxVideoDuration,
+  });
+
+  /// The camera device to default to, if available.
```

Comment:
Might be worth documenting that the default is rear.

Reply:
Done.